content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,121 @@
+
+Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, set up Datadog RUM Browser Monitoring by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK. 
+
+For further instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. 
+
+For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Finding particular Session Replays
+
+Once you're collecting Session Replay data, select **Session Replay available** in the [RUM Explorer][4] to see all sessions with a replay attached to it. From here, you can visualize the data as a **List**, **Timeseries**, or other format. 
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options">}}
+
+### Facets
+
+You can filter Sessions using facets. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+### Views
+
+You are also able to filter by [view][6]. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet and view">}}
+
+## User Journey
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the User Journey by selecting it.
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey">}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**. 
+
+{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal">}}
+
+While hovering over a particular time or view in the User Journey, selecting **Details** opens a side panel where you can examine specific Core Web Vitals of the page, as well as other pertinent information, without leaving the replay.
+
+{{< img src="/getting_started/session_replay/details-panel.png" alt="Panel with additional details">}}
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Developer Tools
+
+Selecting **Dev Tools** enables you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console">}}
+
+You can view the **Resources** and **Traces** associated with a view through using the tabs.
```

Comment:
I would separate these two sentences and have one screenshot per sentence. Also, I would add an anchor link to the APM integration section by using `[word](#apm-integration)`.